<testng-results>
  <reporter-output>
    <line>
      <![CDATA[<br /><div style ="background-color:#EDEDA9;color:green;"><strong>stLaunch-Open Browser and launch Portal URL [PASS]</strong><br />Successfully launch Publishers Page.<br /><strong>Data used:</strong><br />{BrowserType=*firefox, Port=4444, WindowTitle=Experiential Marketing Forum, URL=http://experientialforum.com/, Localhost=localhost}</div>]]>
    </line>
  </reporter-output>
  <suite name="test run">
    <groups>
    </groups>
    <test name="WidgetCreateAccountWithInvalidCredentials">
      <class name="com.sharethis.QAAutomation.createAccountUsingWidget.WidgetCreateAccountWithInvalidCredentials">
        <test-method status="PASS" signature="CloseBrowser()" name="CloseBrowser" is-config="true" duration-ms="104" started-at="2014-05-27T15:08:48Z" finished-at="2014-05-27T15:08:48Z">
        </test-method>
        <test-method status="FAIL" signature="test_WidgetCreateAccountWithInvalidEmail(int, int, int, int, int)" name="test_WidgetCreateAccountWithInvalidEmail" duration-ms="27962" started-at="2014-05-27T15:08:20Z" finished-at="2014-05-27T15:08:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[0]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
	Syntax error, insert "Finally" to complete TryStatement
	Syntax error on token "catch", Identifier expected
	Syntax error on token "}", { expected after this token
	returnVal cannot be resolved
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
	Syntax error, insert "Finally" to complete TryStatement
	Syntax error on token "catch", Identifier expected
	Syntax error on token "}", { expected after this token
	returnVal cannot be resolved

	at com.sharethis.QAAutomation.commonlib.STRegisterAndSign.&lt;init&gt;(STRegisterAndSign.java:424)
	at com.sharethis.QAAutomation.createAccountUsingWidget.WidgetCreateAccountWithInvalidCredentials.test_WidgetCreateAccountWithInvalidEmail(WidgetCreateAccountWithInvalidCredentials.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:580)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:478)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:607)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:874)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:689)
	at org.testng.TestRunner.privateRun(TestRunner.java:566)
	at org.testng.TestRunner.run(TestRunner.java:466)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:301)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:296)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:276)
	at org.testng.SuiteRunner.run(SuiteRunner.java:191)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:808)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:776)
	at org.testng.TestNG.run(TestNG.java:701)
	at org.testng.TestNG.privateMain(TestNG.java:840)
	at org.testng.TestNG.main(TestNG.java:818)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
